src/components/ivdcm_module/src/ivdcm_proxy.cc

```diff
@@ -44,9 +48,56 @@ namespace ivdcm_module {
 
 CREATE_LOGGERPTR_GLOBAL(logger_, "IVDCM")
 
+struct ShowPacketInfo {
+  explicit ShowPacketInfo(const std::vector<uint8_t>* packet)
+    : packet_(packet) {
+  }
+  const std::vector<uint8_t>* packet_;
+};
+std::ostream& operator << (std::ostream& output, const ShowPacketInfo& info) {
```

Comment:
Is it possible not to use streams?
As I mentioned before streams could raise exceptions if you don't disable them explicitly.